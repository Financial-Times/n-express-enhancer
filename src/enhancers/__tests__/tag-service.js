import tagService from '../tag-service';

it('create enhancer to add service to meta of action functions', () => {
	const actionFunction = (param, meta) => meta;
	const enhanced = tagService('some-service')(actionFunction);
	const result = enhanced({});
	expect(result).toMatchSnapshot();
});
