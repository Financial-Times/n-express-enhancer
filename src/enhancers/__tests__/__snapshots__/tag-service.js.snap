// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create enhancer to add service to meta of action functions 1`] = `
Object {
  "service": "some-service",
}
`;
